pkg/minikube/cluster/filesync.go

```diff
@@ -37,6 +38,20 @@ func syncLocalAssets(cr command.Runner) error {
 		return err
 	}
 
+	dirs := map[string]bool{}
+	for _, f := range fs {
+		dirs[f.GetTargetDir()] = true
+	}
+
+	args := []string{"mkdir", "-p"}
```

Comment:
is this gonna work on widnows ? not clear where this is running...inside node or on user machine?

Reply:
This gets executed within the VM.

Reply:
> all our tests for this PR has higher than normal FailureRate....something smells !

Good call. It appears this PR fails if there is nothing to sync!